files/www/cgi-bin/perlfunc.pm

```diff
@@ -1084,25 +1084,25 @@ sub hardware_info
     },
     '0xc302' => {
       'name'            => 'PicoStation2',
-      'comment'         => 'PicoStation2 (NOT M series) in testing',
```

Comment:
the non-M series should not be supported

Reply:
The picostation 2 has 8MB flash and 32Mb RAM.  I thought it did work.  "not M series" because it is the "M" series that is the issue --only have 4MB flash and no way to install.   (all the stuff we support is "M2").

Reply:
It's never been on the SPM.  Only the PS M2

Reply:
OK, let me take it out until we can confirm it works.